web/digital-twin-vis-framework/example-mapbox-vis/README.md

```diff
@@ -114,23 +19,57 @@ A small amount of sample data has been committed to demonstrate the power of the
   - Shows examples of setting up data-driven styling within the `data.json` file.
   - No metadata or timeseries present in this data set.
 - **India**:
-  - Based in India, this data set shows rastered locations of Aqueducts and Canals
+  - Based in India, this data set shows rasterised locations of Aqueducts and Canals.
   - This data is pulled from a public WMS endpoint provided by Stanford University.
   - No metadata or timeseries present in this data set.
 
-In most deployed visualisations, an online stack of microservices will provide data endpoints through which data can be queried/loaded onto the visualisation. In this example, no online stack is used, solely to remove a lengthy prerequisite step. Instead, sample data in GeoJSON files have been added (to be hosted by the visualisation's web server) and, in one case, a community provided WMS endpoint connected to. In production, it is advised that all data is loaded into a geospatial data provider (like GeoServer) and a WMS endpoint used; local files can be utilised but then do not offer the optimisation and caching of services like GeoServer. For more information on how to do this, see the README for the [Stack Data Uploader](https://github.com/cambridge-cares/TheWorldAvatar/tree/main/Deploy/stacks/dynamic/stack-data-uploader).
+In most deployed visualisations, an online stack of microservices will provide data endpoints through which data can be queried/loaded onto the visualisation. In this example, no online stack is used, solely to remove a lengthy prerequisite step. Instead, sample data in local GeoJSON files have been added (to be hosted by the visualisation's web server) and, in one case, a community provided WMS endpoint connected to.
+
+In production, it is advised that all data is loaded into a geospatial data provider (like GeoServer) and a WMS endpoint used; local files can be utilised but then do not offer the optimisation and caching of services like GeoServer. For more information on how to do this, see the README for the [Stack Data Uploader](https://github.com/cambridge-cares/TheWorldAvatar/tree/main/Deploy/stacks/dynamic/stack-data-uploader).
+
+It's also worth noting that with this example visualisation, no stack or FeatureInfoAgent is running, hence these is no support for dynamic metadata or timeseries is available. This is something that we plan to work on in future.
```

Comment:
"hence there is" ... "timeseries available"

Reply:
Updated.